rfb/rfbclient.h

```diff
@@ -296,11 +298,7 @@ typedef struct _rfbClient {
 
 #ifdef LIBVNCSERVER_HAVE_LIBJPEG
 	/** JPEG decoder state. */
-	rfbBool jpegError;
```

Comment:
@dcommander to not break ABI, those should be left in (but clearly marked obsolete and due for removal at some version bump)